tests/ballerina-unit-test/src/test/resources/test-src/task/app-workers.bal

```diff
@@ -2,7 +2,7 @@ import ballerina/io;
 import ballerina/task;
 
 int w1Count;
-error errorW1;
+error? errorW1 = ();
 string errorMsgW1;
```

Comment:
Can we have uninitialized variables in the code with the upcoming changes?

Reply:
This wasn't captured with the df-analysis. I think the point is not initialized vars, its whether or not they're being used without initializing (potential nullpointers)